### PR TITLE
CODEOWNERS

### PR DESCRIPTION
**Pull request by jacopocarlini**
#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by aomegax — APPROVED** [review]
👍

File: CODEOWNERS [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # see https://help.github.com/en/articles/about-code-owners#example-of-a-codeowners-file
 
-* @pagopa-tech
+* @pagopa-tech @jacopocarlini @pasqualespica @aomegax
```
Comment (pasqualespica) [inline CODEOWNERS:3]: Why this changes ?
To manage [pr routing-algorithms](https://docs.github.com/en/enterprise-server@3.2/organizations/organizing-members-into-teams/managing-code-review-settings-for-your-team#routing-algorithms) ?

**Review by pasqualespica — APPROVED** [review]
